src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,10 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod('DELETE')
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ('' === ($requestContent = $request->getContent()) && $request->isMethod('PUT'))
+            || (
+                    '' === ($requestContent = $request->getContent())
+                    && (Request::METHOD_POST === $request->getMethod() || Request::METHOD_PUT === $request->getMethod())
+               )
```

Comment:
May we add some behat tests?

Reply:
Sure, Didn’t do it at first because I wasn’t sure if the mere idea of an empty POST would be accepted.

Reply:
yes looks totally appropriate :)